src/mobile/components/layout/bootstrap.js

```diff
@@ -11,105 +11,118 @@
 // included across most apps and any uncessary bloat should be avoided.
 //
 
-require('jquery');
-const Backbone = require('backbone');
-Backbone.$ = $;
-
-const FastClick = require('fastclick');
-const sd = require('sharify').data;
-const Cookies = require('cookies-js');
-const { parse } = require('url');
-const doc = window.document;
-const CurrentUser = require('../../models/current_user');
-const { init } = require("@sentry/browser");
-const globalReactModules = require('../../../desktop/lib/global_react_modules.tsx');
-const hydrateStitch = require('@artsy/stitch/dist/internal/hydrate').hydrate;
-const { mediator } = require('../../../lib/mediator');
-const syncAuth = require('../../../lib/syncAuth.ts').default;
-const { logoutEventHandler } = require('../../../desktop/lib/deprecated_global_client_setup.tsx');
-
-export const bootstrap = function() {
+require("jquery")
```

Comment:
so much changes to this file. all i did was to add loadSegment call.

Reply:
the conversion was done by linter. have cut a [separate pr](https://github.com/artsy/force/pull/8983) for the conversion. once that pr is merged i'll rebase, then the change here will only be adding `loadSegment`.

Reply:
done.